### ISSUE TITLE
No Sensu Go for Debian?

### ISSUE THREAD

**Issue by samitks** [sensu/sensu-docs#1006]
I checked the complete documentation and found that Sensu Go can be installed only on Ubuntu. I checked the repo on cloudpackage.io and found that there are no Sensu Go packages for Debian platform. 

Can you please confirm that you are not supporting or maintaining debian packages? In that case, what about the uses of Sensu Go who uses debian servers in production?

**Comment by samitks** [comment]
I could see that there are some packages in https://packagecloud.io/sensu/nightly with name sensu-backend and sensu-agent. Are these can be used as an alternative for the Sensu-Go in Debian or these are completely different? 
Also, what kind of packages are kept in this 'nightly' repo?

**Comment by apaskulin** [comment]
Thanks for your questions, @compengsamit! The nightly repo is not currently being used, but we hope to bring it back soon.

To answer your questions about Debian as a supported platform, we are planning to provide Debian packages for Sensu Go in a future release. Until then, you can use the Ubuntu packages with a slightly modified installation process.

Add the Sensu repository for Ubuntu:

```
curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
```

And run the following two commands:

```
sudo sed -i 's/debian\/ stretch/ubuntu\/ xenial/g' /etc/apt/sources.list.d/sensu_stable.list

sudo apt-get update
```

Then you can install packages following the instructions in the installation guide https://docs.sensu.io/sensu-go/5.0/installation/install-sensu/

Another option is to use the binary artifact available for all Linux distros, available here https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.0.0/sensu-go-5.0.0-linux-amd64.tar.gz. I’ll update this issue when the Debian packages are available.

**Comment by samitks** [comment]
Thanks @apaskulin 
I used the deb packages of Ubuntu/xenial and was able to install those packages on Debian 9.

**Comment by apaskulin** [comment]
Hi @compengsamit! Just wanted to let you know that Sensu Go packages are available for Debian as of version 5.2.
